components/icons/images/tier2/classes.svg

```diff
@@ -0,0 +1,181 @@
+              </g>
+            </g>
+          </g>
+        </g>
+        <g id="Rectangle_Clipped-13" data-name="Rectangle Clipped-13">
+          <g id="path-13-2">
+            <rect id="Rectangle_489" data-name="Rectangle 489" width="6.545" height="6.545" rx="1.091" transform="translate(149.199 184.007)" fill="none"/>
+          </g>
+          <g id="Group_899" data-name="Group 899">
+            <g id="Group_898" data-name="Group 898" clip-path="url(#clip-path-7)">
+              <g id="Rectangle_Clipped-14" data-name="Rectangle Clipped-14">
+                <g id="Rectangle-7">
+                  <g id="path-13-3">
+                    <path id="Path_329" data-name="Path 329" d="M154.654,192.735H150.29a3.276,3.276,0,0,1-3.273-3.273V185.1a3.276,3.276,0,0,1,3.273-3.273h4.364a3.276,3.276,0,0,1,3.272,3.273v4.364A3.276,3.276,0,0,1,154.654,192.735Zm-3.273-4.364h2.182v-2.182h-2.182Z" fill="#494c4e"/>
+                  </g>
+                </g>
+              </g>
+            </g>
+          </g>
+        </g>
+        <g id="Rectangle_Clipped-15" data-name="Rectangle Clipped-15">
+          <g id="path-15-2">
+            <rect id="Rectangle_491" data-name="Rectangle 491" width="6.545" height="6.545" rx="1.091" transform="translate(157.927 184.007)" fill="none"/>
+          </g>
+          <g id="Group_901" data-name="Group 901">
+            <g id="Group_900" data-name="Group 900" clip-path="url(#clip-path-8)">
+              <g id="Rectangle_Clipped-16" data-name="Rectangle Clipped-16">
+                <g id="Rectangle-8">
+                  <g id="path-15-3">
+                    <path id="Path_330" data-name="Path 330" d="M163.381,192.735h-4.364a3.276,3.276,0,0,1-3.272-3.273V185.1a3.276,3.276,0,0,1,3.272-3.273h4.364a3.276,3.276,0,0,1,3.273,3.273v4.364A3.276,3.276,0,0,1,163.381,192.735Zm-3.273-4.364h2.182v-2.182h-2.182Z" fill="#494c4e"/>
+                  </g>
+                </g>
+              </g>
+            </g>
+          </g>
+        </g>
+        <g id="Rectangle_Clipped-17" data-name="Rectangle Clipped-17">
+          <g id="path-17-2">
+            <rect id="Rectangle_493" data-name="Rectangle 493" width="6.545" height="6.545" rx="1.091" transform="translate(166.654 184.007)" fill="none"/>
+          </g>
+          <g id="Group_903" data-name="Group 903">
+            <g id="Group_902" data-name="Group 902" clip-path="url(#clip-path-9)">
+              <g id="Rectangle_Clipped-18" data-name="Rectangle Clipped-18">
+                <g id="Rectangle-9">
+                  <g id="path-17-3">
+                    <path id="Path_331" data-name="Path 331" d="M172.108,192.735h-4.363a3.276,3.276,0,0,1-3.273-3.273V185.1a3.276,3.276,0,0,1,3.273-3.273h4.363a3.276,3.276,0,0,1,3.273,3.273v4.364A3.276,3.276,0,0,1,172.108,192.735Zm-3.272-4.364h2.181v-2.182h-2.181Z" fill="#494c4e"/>
+                  </g>
+                </g>
+              </g>
+            </g>
+          </g>
+        </g>
+      </g>
+    </g>
+  </g>
+</svg>
```

Comment:
This is pretty huge! Make sure you [follow the instructions](https://github.com/BrightspaceUI/core/tree/master/components/icons#updating-or-contributing-new-icons) for contributing. I have a feeling after putting this through SVGOmg it'll look a little better (no need for lots of `id`s and `data-name`s.

Reply:
So I tried to run it through SVGOMG but it doesn't seem to help much or at all 😞

Reply:
Haha I see that. I don't think any of the `data-name` attributes are needed, so you could try removing them. Also likely most of the `id`s aren't needed either. You could search for each one and if it's only referenced once, then remove it.

After that, re-run it through SVGOmg and it may be able to collapse a lot of the `<g>`s together. My guess is whatever editor was used to make this is leaving artifacts in the SVG for itself. You could also ask Nick if maybe there's a different way it could be exported to exclude all that.

@geurts I'm not sure if you've found a good technique to get cleaner SVGs out of these apps.

Reply:
So I removed `date-name` and `id` attributes wherever I could. I'm wondering if it's possible to replace the `clipPaths` with something else 🤔

Reply:
Agreed, those seem like overkill for something that's literally 9 boxes. I wonder if the tier3 one (which is just a single path) could just be tweaked to be slightly smaller.

Reply:
I actually recreated this back in April for Wave, I guess it got lost in the shuffle! I'll attach it here. When I hack this into Wave via Chrome dev tools, it works well and looks good 👍 

Oh, GitHub won't let me drop an SVG in here. I'll send it over Slack.